apps/ledger-live-desktop/package.json

```diff
@@ -40,6 +40,7 @@
     "release": "node ./tools/dist --release -v",
     "test": "pnpm test:jest && pnpm test:playwright",
     "test:codecheck": "node ./tools/ci.js",
+    "jest": "jest",
```

Comment:
Is this needed? @henrily-ledger

Reply:
I'll remove it, It was just a easier for me to test just one specific file with `pnpm desktop jest <filename>`